docs/getting-started.md

```diff
@@ -7,22 +7,29 @@ description: Jump in and get started quickly using Momento Serverless Cache with
 ---
 
 # Getting Started with Momento Serverless Cache
-If you're looking to jump in and get started with Momento Serverless Cache with an SDK or CLI, you've come to the right place. There are multiple ways to access the service below. 
+If you're looking to jump in and get started with Momento Serverless Cache with an SDK or CLI, you've come to the right place. Want to quickly get a feel for the Momento experience without installing any software? You can do so right here [in your browser](#try-our-cli-and-an-sdk-in-your-browser). Or, if you're ready to setup your development environment and get to building your own applications with Momento you can follow the [simple instructions below](#install-the-momento-command-line-interface-cli).
 
-## Try our CLI in your browser
+## Try our CLI and an SDK in your browser
 
-You can try out our CLI tool today, right in your browser.  No software installation required!
+You can try out Momento Serverless Cache today, right in your browser. No software installation required! **Note: Momento Serverless Cache is an API based serverless service. It does not deploy anything into your (AWS, GCP, Azure, etc.) account.**
 
-Head over to our [Try Momento Serverless Cache page](https://www.gomomento.com/try-momento-for-free), and
-you can experiment with Momento CLI commands in a shell session directly inside your browser.
-You can create a free auth token, create a cache, and start running `set` and `get` commands
-on your cache. It only takes a few minutes!
+First, request your free auth token, create a cache, configure your CLI, and start running `set` and `get` commands on your cache. Launch below.
```

Comment:
This list seems out of order to me. Shouldn't it be 
```markdown
...free auth token, configure the CLI, create a cache, and start running `set` and `get` commands...
```
Since the image you're displaying directly underneath this message has a launch button on it, I'm not sure `Launch below` is necessary

Reply:
Hmm. Normally I think the order I listed is correct. In this case we're letting the CLI configure create the default cache for us. I think the order is still reasonable - I started the configure first, and the cache was created subsequent to this.

Agree the "Launch below" is not strictly necessary, but it does make things clearer and I don't want to lose anyone in the flow. You'd be amazed how people find ways to get lost! Suspect Hannah might take a pass at this and clean it all up anyway - maybe reorganize the pages and the menu as you'd suggested.